debian/libpam-policycache.postinst

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+DEFAULT_CACHE_STORAGE='/var/cache/libpam-policycache'
+
+case "$1" in
+  configure)
+    chmod 0700 "$DEFAULT_CACHE_STORAGE"
+    chown root:root /usr/bin/pam-escalate-helper
+    chmod 6755 /usr/bin/pam-escalate-helper
```

Comment:
I suppose for the goal of file ownership you could also setgid $DEFAULT_CACHE_STORAGE, the directory? If you're already assuming that you need to run as root:root anyway (because unprivileged users won't work with PAM - which I'm still curious about), then I guess hardcoding the setgid to 0 wouldn't hurt. But I defer to you because I'm a little clueless around the proper setuid/seteuid/setgid/setegid handling here, unfortunately. It just stood out to me that noone else seems to do this.